src/features/game/events/index.ts

```diff
@@ -189,6 +191,7 @@ export const PLAYING_EVENTS: Handlers<PlayingEvent> = {
   "bean.bought": beanBought,
   "cropReward.collected": collectCropReward,
   "treeReward.collected": collectTreeReward,
+  "fruit.planted": plantFruit,
```

Comment:
dont we need a harvest action for the fruit too?

i assume it will use the fruit basket.

Reply:
Yes we do, but it'll come on a next PR.